migrate-configuration-v2.rst

```diff
@@ -7,26 +7,36 @@
 
-We are announcing the **deprecation of building without a configuration file** (``.readtehdocs.yaml``)
-together with the deprecation of configuration file v1 as well.
+We are announcing a new **requirement for all builds to use our configuration file version 2: ``.readthedocs.yaml``**.
+This announcement deprecates builds without a configuration file, as well as version 1 of our configuration file.
 
-Read the Docs *will soon start requiring* a ``.readthedocs.yaml`` configuration file
+Read the Docs *will start requiring* a ``.readthedocs.yaml`` configuration file
 for all projects in order to build documentation successfully.
-We will stop creating a default configuration file *behind the scenes*, making assumptions for your projects,
-and automatically fixing problems with project configurations.
+We will stop supporting builds without explicit configuration,
+because this creates implicit dependencies that users aren't aware of.
 
-We plan to deploy these changes and **start failing builds without a configuration file in September 2023**.
+**We plan to start failing builds not using configuration file version 2 on September 4, 2023**.
 This matches the period when ``urllib3`` `drops support for OpenSSL 1.1.1 <https://github.com/urllib3/urllib3/issues/2168>`_
 which is the OpenSSL version installed by the build image used when the project doesn't specify a configuration file.
 
 The main reason behind this decision is that it's hard to maintain a set of defaults that are useful
-for all of our users and the different tools and libraries they use to build documentation.
-Without a configuration file, we must guess at the configuration the project needs to build and this is also difficult to do reliably.
+for all of our users and the different tools they use to build documentation.
+Without a configuration file, we must guess the configuration the project needs to build, which is difficult to do reliably.
 Even if the defaults we chose worked well for a period of time,
-we found them impossible to change over time.
+we found it impossible to change over time.
 Doing so would break projects that were relying on our defaults instead of specifying a configuration file
 and pinning their dependencies (e.g. OS, Python version, dependencies, etc).
 
+Deprecation timeline
+--------------------
+
+We understand this change will effect many of our users,
+so we have a timeline to communicate this deprecation to our users effectively.
+
+* **June 2023**: Email project owners with unsupported configurations once a week, along with an onsite notification to alert them of the deprecation.
+* **Monday, July 24, 2023**: Do the first brownout (temporarily enforce this deprecation) for 2 hours in 3 timezones, 9:00-11:00, in 3 timezones: UTC-8, UTC, and UTC+8.  
+* **Monday, August 7, 2023**: Do a second brownout (temporarily enforce this deprecation) for 2 hours in 3 timezones, 9:00-11:00, in 3 timezones: UTC-8, UTC, and UTC+8.  
```

Comment:
Brought up in comments here, but maybe lets make this even easier on us and replicate what GitHub did: 12h window first date, 24h window the second date. I'm guessing it might be hard to have folks around for all of these 2h windows, thinking about this a bit more. We'll definitely catch more projects this way too.

Reply:
If we start a 12h window Mon Sept 4 @ 0:00 PT and go till 12:00 PT, this is what the overlap looks like:

![image](https://github.com/readthedocs/blog/assets/1140183/680af644-414e-4f2e-abba-c3c2da0f9d0a)

Asia users will get errors starting at the end of their day and the errors will resolve overnight, so not *great*, but we have more users in EU/US anyways.

Reply:
Sounds good to me 👍